src/header.rs

```diff
@@ -293,7 +293,7 @@ impl<W: io::Write> UncompressedHeader for BitWriter<W, BigEndian> {
       assert!(!fi.sequence.decoder_model_info_present_flag);
       assert_eq!(fi.sequence.operating_points_cnt_minus_1, 0);
       assert_eq!(fi.sequence.operating_point_idc[0], 0);
-      self.write(5, fi.sequence.level_idx[0])?; // level
+      self.write(5, 31)?; // level
```

Comment:
why is it 31?

Reply:
avif used to always use level 31, guess it doesn't anymore